pkgs/applications/radio/sdrangel/default.nix

```diff
@@ -0,0 +1,81 @@
+{ stdenv, mkDerivation, fetchFromGitHub,
+cmake, pkgconfig, glew, opencv3, libusb, libopus,
+boost, qtbase, qtwebsockets, qtmultimedia, wrapQtAppsHook,
+fftw, fftwFloat,
+limesuite,
+ocl-icd,
+libpulseaudio, libav, libiio
+}:
+
+let
+
+  codec2 = stdenv.mkDerivation {
+    name = "codec2";
+    src = fetchFromGitHub {
+      owner = "drowe67";
+      repo = "codec2";
+      rev = "567346818c0d4d697773cf66d925fdb031e15668";
+      sha256 = "0ngqlh2cw5grx2lg7xj8baz6p55gfhq4caggxkb4pxlg817pwbpa";
+    };
+    nativeBuildInputs = [ cmake ];
+  };
+
+  cm256cc = stdenv.mkDerivation {
+    name = "cm256cc";
+    src = fetchFromGitHub {
+      owner = "f4exb";
+      repo = "cm256cc";
+      rev = "f21e8bc1e9afdb0b28672743dcec111aec1d32d9";
+      sha256 = "1imxg9hxh92r8ip161980axavvx1sr321lw0g8rbza59s50sd17g";
+    };
+    nativeBuildInputs = [ cmake ];
+    buildInputs = [ boost ];
+  };
+
+  serialdv = stdenv.mkDerivation {
+    name = "serialdv";
+    src = fetchFromGitHub {
+      owner = "f4exb";
+      repo = "serialdv";
+      rev = "d5830fae715f4acffa2acec33539c3a11c17a1c9";
+      sha256 = "0swalyp8cbs7if6gxbcl7wf83ml8ch3k7ww4hws89rzpjvf070fr";
+    };
+    nativeBuildInputs = [ cmake ];
+  };
```

Comment:
Why not add all of these dependencies in separate PRs and as separate packages? It'll probably easier for everyone to review these and it'll be easier for you to package sdrangel.

Reply:
That's a lot of PRs (and thus wait) and those dependencies are from the same dev as sdrangel so my guess is they probably won't be used anywhere else

Reply:
I agree it should take some patience if you add them all but I think this is more consistent with the workflow around new packages. I'm myself a somewhat experienced new packages contributor as well and I've had such iteration my self which took some time because of dependencies by the same developer which were packaged separately. See https://github.com/NixOS/nixpkgs/pull/64570, https://github.com/NixOS/nixpkgs/pull/64453, and https://github.com/NixOS/nixpkgs/pull/65480 .

I think it's worth it because it's easy for the maintainers with the right permissions to review you.

Reply:
Ok i'll do that then, i'll have to wait for the other PR to get agreed on before finishing that one then ^

Reply:
Also that's kinda annoying because i'm not in the maintainers list yet ^